### DISCUSSION TITLE
C Programlama İşaretçiler (Pointers) | SonsuzUs Github

### DISCUSSION THREAD

**Discussion by giscus** [sonsuzus/sonsuzus.github.io#198] [Site]
# C Programlama İşaretçiler (Pointers) | SonsuzUs Github

İşaretçiler, bilgisayar belleğindeki belli bir adresin, genellikle bir değişkene ait, kaydedildiği ve kaydedilen bellek adresine doğrudan erişim sağlayan değişkenlerdir.

https://sonsuzus.github.io/posts/cpp-isaretciler-pointers

**Comment by sonsuzus** [comment]
İşaretçi soruları oldukça önemli.
